### DISCUSSION TITLE
Type checking mode toggle

### DISCUSSION THREAD

**Discussion by jadeidev** [microsoft/pylance-release#2651] [Enhancement]
Ability to quickly change between the off\simple\strict type checkers instead of going to settings. perhaps at the status bar  
![image](https://user-images.githubusercontent.com/32917209/134428733-20b1cf53-88c9-400a-9611-2c642fbd081d.png)

**Comment by jakebailey** [comment]
Thanks for the suggestion; we had actually been talking about this specific thing just yesterday. 🙂

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by bschnurr** [comment]
New language status bar item.

https://devblogs.microsoft.com/python/python-in-visual-studio-code-october-2022-release/#new-type-checking-status-button-for-pylance

note just for switching between basic and off

**Reply by jadeidev** [reply]
Thank you for this
